lib/repl.ts

```diff
@@ -42,6 +42,17 @@ export const Repl = new class {
 		if (!process.listeners('SIGINT').length) {
 			process.once('SIGINT', () => process.exit(128 + 2));
 		}
+		(global as any).heapdump = (targetPath?: string) => {
+			if (!targetPath) targetPath = `${filename}-${Date.now()}`;
```

Comment:
```suggestion
			if (!targetPath) targetPath = `${filename}-${new Date().toISOString()}.heapsnapshot`;
```

Reply:
(heapsnapshot is automatically appended)

Reply:
(edited)